app/controllers/settings/from_address_controller.rb

```diff
@@ -1,14 +1,22 @@
 class Settings::FromAddressController < FormController
   before_action :assign_from_address
 
-  def index; end
+  def index
+    @from_address_presenter = FromAddressPresenter.new(@from_address)
```

Comment:
if we are already assigning the presenter in the before_action we dont need to do it again here